docs/source/getting_started/index.rst

```diff
@@ -53,74 +54,75 @@ have created before, and expose ports for communication:
   -e POSTGRES_PASSWORD=mysecretpassword \
   -p 5432:5432 \
   --network=iroha-network \
-  -d postgres:9.5
+  -d postgres:9.5 \
+  -c 'max_prepared_transactions=100'
 
 .. note:: If you already have Postgres running on a host system on default port
   (5432), then you should pick another free port that will be occupied. For
   example, 5433: ``-p 5433:5432 \``
 
 Creating Blockstore
 ^^^^^^^^^^^^^^^^^^^
-Before we run Iroha container, we should create persistent volume to store
+Before we run Iroha container, we may create a persistent volume to store
 files, storing blocks for the chain. It is done via the following command:
 
 .. code-block:: shell
 
   docker volume create blockstore
 
-Configuring Iroha Network
-^^^^^^^^^^^^^^^^^^^^^^^^^
+Preparing the configuration files
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 .. note:: To keep things simple, in this guide we will create a network
-  containing only one node. To understand how to run several peers, follow
-  `this guide. <github.com/hyperledger/iroha/tree/master/deploy/ansible>`_
+  containing only a single node. To understand how to run several peers, follow
+  :ref:`deploy-guide`
 
 Now we need to configure our Iroha network. This includes creating a
 configuration file, generating keypairs for a users, writing a list of peers
-and creating a genesis block. However, we have prepared an example
-configuration for this guide, so you can start playing with Iroha faster.
-In order to get those files, you need to clone the
-`Iroha repository <github.com/hyperledger/iroha>`_ from Github.
+and creating a genesis block.
+
+Don't be scared away — we have prepared an example configuration for this guide, so you can start testing Iroha node now.
+In order to get those files, you need to clone the `Iroha repository <github.com/hyperledger/iroha>`_ from Github or copy them manually (cloning is faster, though).
 
 .. code-block:: shell
 
   git clone -b master https://github.com/hyperledger/iroha --depth=1
 
-.. hint:: ``--depth=1`` option allows us to download only latest commit and
+.. hint:: ``--depth=1`` option allows us to download only the latest commit and
   save some time and bandwidth. If you want to get a full commit history, you
   can omit this option.
 
+There is a guide on how to set up the parameters and tune them with respect to your environment and load expectations: :ref:`configuration`.
+We don't need to do this at the moment.
+
 Starting Iroha Container
 ^^^^^^^^^^^^^^^^^^^^^^^^
-We are ready to launch our Iroha container. Let's do it with the following
-command
+We are almost ready to launch our Iroha container.
+You just need to know the path to configuration files (from the step above).
+
+Let's start Iroha node in Docker container via the following command:
 
 .. code-block:: shell
 
-  docker run -it --name iroha \
+  docker run --name iroha \
   -p 50051:50051 \
-  -v $(pwd)/iroha/example:/opt/iroha_data \
+  -v YOUR_PATH_TO_CONF_FILES:/opt/iroha_data \
```

Comment:
not a good idea to change that line. The reason is - the reader is not able any more do just copy-paste :( 

I suggest to revert that line change and still provide an explanation in a section below.

Reply:
true - that's why I added the example below (within the instructions). I think we could leave the option of custom directory somewhere too. Maybe better to leave the previous version here and explain that a custom directory could be used instead further in the instructions?